sigh/lib/sigh/runner.rb

```diff
@@ -216,7 +216,12 @@ def certificate_to_use
     # Downloads and stores the provisioning profile
     def download_profile(profile)
       UI.important "Downloading provisioning profile..."
-      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}.mobileprovision" # default name
+      profile_name ||= "#{profile_type.pretty_type}_#{Sigh.config[:app_identifier]}"
```

Comment:
can we append the .mobileprovision directly in the `|| =`  above?
as right now it seems to be appended anyway 

the prev. implementation also was not 100% right, i think